### PR TITLE
Image Upload

### PR DESCRIPTION
**Pull request by SethDonohue**
Added Camera Route
Added SendPicture Route
Added Basic Results

### CODE REVIEW

File: src/routes/send-picture.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    //        too large to send and/or takes too long.
+    // TODO: Seth - make the resize options variables that can be set with environment variables.
+    // TODO: Seth - Trigger the load state when the image is sent to show a loading graphic.
+    ImageManipulator.manipulate(this.state.image.uri, [{resize: {width: 500}}], { compress: 0.5 })
+    .then((response) => {
+      resizedImage = response;
+      ImageStore.getBase64ForTag(
+        resizedImage.uri,
+        // getbase64ForTag SUCCESS callback
+        base64String => {
```
Comment (AndyCErnst) [inline src/routes/send-picture.js:26]: this is a big function for inline, it should be defined outside of here for clarity

File: src/routes/send-picture.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            onPress={this.sendPicture}
+          />
+
+          <Image
+          style={styles.image}
+          resizeMode={'contain'}
+          source={this.state.image} />
+          <Button
+            title='Re-take Picture'
+            onPress={() => goBack()}
```
Comment (AndyCErnst) [inline src/routes/send-picture.js:85]: could just be `onPress={goBack}`

Comment (SethDonohue) [reply src/routes/send-picture.js:85]: Not sure hwy, but this does not work as you suggest, same as in expo-camera route, it does not work as just {goBack}, needs to be an anonymous function for some reason.

File: src/routes/send-picture.js (hunk 1 above)
Comment (AndyCErnst) [reply src/routes/send-picture.js:26]: would still be good to refactor this to be clearer, but HACKATHON.
Approving this PR

**Review by AndyCErnst — APPROVED** [review]

**Pull request merged**